.github/.OwlBot.yaml

```diff
@@ -353,7 +353,7 @@ deep-remove-regex:
   - /internal/generated/snippets/shopping/css/apiv1/
   - /internal/generated/snippets/shopping/merchant/quota/apiv1beta/
   - /internal/generated/snippets/shopping/merchant/inventories/apiv1beta/
-  - /internal/generated/snippets/shopping/merchant/reports/apiv1beta/
+  - /internal/generated/snippets/shopping/merchant/lfp/apiv1beta/
```

Comment:
I think you meant to copy the line and change it there rather than in place :)

Reply:
yup, thanks!